fastpass/src/main/scala/scala/meta/internal/fastpass/pantsbuild/BloopPants.scala

```diff
@@ -381,15 +381,8 @@ private class BloopPants(
 
   def getSources(target: PantsTarget): List[Path] = {
     def single(t: PantsTarget): List[Path] =
```

Comment:
I'm not sure I get why this function was called single. Do you have a suggestion for a new name, if required?

Reply:
It's a bad name, it's called "single" because it handles a single pants target and `getSources` may need to handle multiple targets. I don't have any better suggestion however, maybe `sourcesForSingleTarget`?